doorway-ui-components/src/blocks/ImageCard.scss

```diff
@@ -4,7 +4,7 @@
 .image-card {
   /* Component Variables */
   --default-background-color: var(--bloom-color-gray-500);
-  --border-radius: 0px;
+  --border-radius: 6px;
```

Comment:
`--bloom-rounded-md` or `lg` perhaps? (neither is exactly 6px but both are close --whichever looks better)